src/store/wallet/wallet-epics.ts

```diff
@@ -65,7 +65,12 @@ const addPendingSwapTransactionEpic: Epic<Action> = action$ =>
                     payload.bocHash
                 )
             ).pipe(
-                map(() => addPendingSwapTransactionActions.success()),
+                concatMap(() => [
+                    addPendingSwapTransactionActions.success(),
+                    loadBalancesActions.submit(
+                        Address.parse(payload.senderRawAddress).toString()
```

Comment:
looks like redundant parsing and stringifying.  
would this action work if you will pass raw address in it?